src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -209,13 +213,13 @@ private int compareLargestFirst(Queue a, Queue b) {
     }
 
 
-    private static class GetQueueStatsRequest<CTX> {
+    static class GetQueueStatsRequest<CTX> {
         private CTX mCtx;
         private GetQueueStatsMentor<CTX> mentor;
         private List<String> queueNames;
         /* TODO: Why is 'queuesJsonArr' never accessed? Isn't this the reason of our class in the first place? */
         private JsonArray queuesJsonArr;
-        private List<Queue> queues;
+        List<Queue> queues;
```

Comment:
Do not change visibility of methods/variables just to test them. This is bad practice. If you can find a workaround or there is a arguable reason, it may be ok.